tripl/tripl.py

```diff
@@ -541,37 +546,36 @@ def loads(cls, filenames, schema=None, id_attrs=None):
         return result
 
     def dump(self, filename):
-        "Save semantic graph to a json file as an EAV index."
+        """Save semantic graph to a json file as an EAV index."""
         with open(filename, 'w') as fp:
             json.dump(self._eav_index.to_dict(), fp, default=list)
 
-
     # # Now our query engine
     # We have a few different kind of queries we want to be able to execute
 
     # Match pattern
     # simple
-    #{'cft:id': 'dk398fjd03kjdfkj23'}
+    # {'cft:id': 'dk398fjd03kjdfkj23'}
     # more interesting
-    #{'cft:dataset': {'cft.dataset:id': 'whatever-crazy-id'}}
+    # {'cft:dataset': {'cft.dataset:id': 'whatever-crazy-id'}}
     # This can more or less always be done efficiently very easily, and extends on base GraphQL, om-next etc
     # significantly;
 
     # My python Datalog grammar proposal:
-    ##base query; matches graph
-    #{'find': ['?x', '?y'],
```

Comment:
Please un-nuke this indentation

Reply:
Sorry, it appears PyCharm had trouble unindenting `  #x` into `#  x`.